presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,83 @@
+=============
+Release 0.239
+=============
+
+**Highlights**
+==============
+* Add support for ``DEFINER`` and ``INVOKER`` view security models.
+* Add support for caching Glue metastore in Hive connector.
+* Add Pinot SQL endpoint support.
+
+**Details**
+==============
+
+General Changes
+_______________
+* Fix incorrect results from :func:`classification_miss_rate`, :func:`classification_fall_out`
+  and :func:`classification_precision` (:pr:14740).
```

Comment:
I think you need
```
:pr:`14740`
```
Could you visually check the content of the html by running.
```
cd presto-docs
make clean
make html
```